crt_portal/static/sass/custom/typography.scss

```diff
@@ -17,12 +17,11 @@ h3 {
 }
 
 // TODO (ARS): Don't target raw HTML elements with styles.
```

Comment:
I bet there's a way I can fix this in my next PR by adjusting our USWDS settings rather than using an override.